torch_xla/csrc/tensor_methods.cpp

```diff
@@ -385,6 +386,29 @@ XLATensor XLATensor::get_dimensions_size(const XLATensor& input,
                           at::ScalarType::Int);
 }
 
+void XLATensor::sgd_optimizer_step(const XLATensor& found_inf, XLATensor& step,
```

Comment:
Nit, Let's group `step`, `param` and `buf` in the front since `sgd_optimizer_step` is actually an inplace operations. It is more clear to put all tensors you want to update in the front and that's what we have done for most of our inplace ops.

I was thinking about renaming this function to `sgd_optimizer_step_` since it is an inplace op but I guess `step` is an indication of inplace update so it is fine.